core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts

```diff
@@ -17,56 +17,99 @@
  * fields.
  */
 
+export class StateAndContentContainer {
+  private stateName: string;
+  private contentID: string;
+  private contentText: string;
+
+  public getStateName(): string {
+    return this.stateName;
+  }
+
+  public setStateName(stateName: string): void {
+    this.stateName = stateName;
+  }
+
+  public getContentID(): string {
+    return this.contentID;
+  }
+
+  public setContentID(contentID: string): void {
+    this.contentID = contentID;
+  }
+
+  public getContentText(): string {
+    return this.contentText;
+  }
+
+  public setContentText(contentText: string): void {
+    this.contentText = contentText;
+  }
+
+  constructor(stateName: string, contentID: string, contentText: string) {
+    this.setStateName(stateName);
+    this.setContentID(contentID);
+    this.setContentText(contentText);
+  }
+}
+
```

Comment:
Sorry for the wrong link earlier: https://www.typescriptlang.org/docs/handbook/classes.html#parameter-properties. We can declare the parameter as `private readonly` in the constructor itself and we don't need to set the variable in the constructor nor declare it in the class.

We can also use TypeScript's `get` and `set` keywords in the getter/setter methods: https://www.typescriptlang.org/docs/handbook/classes.html#accessors.

Also note that TypeScript's methods are public be default, so the public annotation isn't strictly necessary.